### DISCUSSION TITLE
Column mode (Alt+Shift feature from Notepad++)

### DISCUSSION THREAD

**Discussion by thedeadfish59** [0x7c13/Notepads#1171] [Ideas]
Hi, is it possible to implement this feature?

**Comment by 0x7c13** [comment]
No, it would require WinRT RichEdit control to support it first (Meaning Microsoft needs to implement this feature.)

**Reply by zenvent** [reply]
Would this include multi cursor support in general? That's really the only big feature I can't seem to find.

**Reply by 0x7c13** [reply]
This also requires underline RichEdit component to implement/realize.
